### ISSUE TITLE
Mixins were removed per issue #14 and added to end of repo as a squad lab

### ISSUE THREAD

**Issue by MicFin** [ga-wdi-boston/css_layout_with_bootstrap#27]
Mixins were removed as presented material per issue #14.  Since the repository presentation went quickly without mixins, we let the squads work together to read the Additional Resource in their group and implement the examples.
https://coderwall.com/p/wixovg/bootstrap-without-all-the-debt

Issue to remove mixins:
https://github.com/ga-wdi-boston/html-css-layout-bootstrap-sass/issues/14

**Comment by Jcornmanhomonoff** [comment]
@MicFin do I need to change the material to include this as a lab at the end?

**Comment by jrhorn424** [comment]
@MicFin Did we discuss this in 016? Are you suggesting that this come back into the README as a Lab? I think that's appropriate. Prior to #14, this exercise was included as a code-along, and it took too long. It could be a nice challenge at the end of the talk, though.

**Comment by jrhorn424** [comment]
@MicFin Let us all know why you are assigning yourself in the future? Thanks. I typically assign myself when I'm actively working on something. I've done it as a reminder to myself in the past, but I'm not sure that's a great idea.

**Comment by MicFin** [comment]
@jrhorn424 You're right, I was using it as a reminder to respond to Jessica.  In my "queue".  I'm also not sure that's the best way but if I can't respond immediately to a comment and I lose the alert then I worry it would get lost in the shuffle.  

I don't think WDI developers use SASS mixins and it adds a layer of complexity that might only confuse more people than it helps.  However, I also think that `extend` is useful and more straight forward.  This is another good resource on `extend` 
https://stackoverflow.com/questions/30744625/sass-and-bootstrap-mixins-vs-extends

I also think we it would be fine without `mixins` or `extend`.  They could both remain as additional resources.
